sftplib/src/test/java/com/freiheit/fuava/sftp/PseudoSFTPTest.java

```diff
@@ -100,13 +105,27 @@ public String getControlFileEnding() {
             assertIsNullOrEmpty( finalState, "/skipped/" + LocalDate.now().format( DateTimeFormatter.BASIC_ISO_DATE ) );
             assertIsNullOrEmpty( finalState, "/processed" );
             assertIsNullOrEmpty( finalState, "/archived" );
+
+            final String incomingDirPath = "/incoming/";
+            final TestFolder<String> testFolderIncoming = finalState.get( incomingDirPath );
+            Assert.assertNotNull( testFolderIncoming, "Incoming directory '" + incomingDirPath + "' should not be null" );
+            final Set<String> incomingContent = testFolderIncoming.getItemKeys();
+            Assert.assertEquals( incomingContent.size(), 0, "Incoming directory should be empty." );
+            System.out.println( incomingContent );
```

Comment:
We should not use `System.out` in tests (I know you are not the first to do this...)